src/main/resources/issues/publish-parameters.jelly

```diff
@@ -27,6 +27,9 @@
   <f:entry title="${%title.referenceJob}" field="referenceJobName" description="${%description.referenceJob}">
     <f:combobox/>
   </f:entry>
+  <f:entry title="${%title.referenceBuildId}" field="referenceBuildId" description="${%description.referenceBuildId}">
```

Comment:
I think a UI configuration makes no sense, or do you use the snippet generator on your own for this type of configuration? So it would be save to remove these lines.

Reply:
Removed.